CONTRIBUTING.rst

```diff
@@ -97,8 +97,8 @@ Code of Conduct
 ---------------
 
 This project adheres to the Contributor Covenant `code of
-conduct <CODE_OF_CONDUCT.rst>`__. By participating, you are expected to
-uphold this code. Please report unacceptable behavior at
+conduct <https://graphql-compiler.readthedocs.io/en/latest/about/code_of_conduct.html>`__. By
```

Comment:
I changed this link because it won't work from ReadTheDocs as is. I verified that it works.